services/QuillLMS/app/workers/send_segment_identify_call_for_all_admins_worker.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SendSegmentIdentifyCallForAllAdminsWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::LOW
+
+  def perform
+    user_ids = User.where(role: User::ADMIN).ids
```

Comment:
While we already have [scopes](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/user.rb#L216) for :teacher and :student you could add one for admin:  

```  scope :admin, ->  { where(role: ADMIN) }```

Then this perform method becomes a one liner:
```
User.admin.ids { |user_id|  IdentifyWorker.perform_async(user_id) }
```